packages/components/src/drop-zone/animators/maybe-fade.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import type { ReactNode } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+import { useReducedMotion } from '@wordpress/compose';
+
+/**
+ * Handles fade-in/fade-out animation for its children if `reducedMotion` is disabled,
+ * if not, just shows/hides the children without any animation.
+ *
+ * Relies on the animations being defined in CSS styles.
+ */
+export const MaybeFade: React.FC< {
+	show: boolean;
+	children: ReactNode;
+	duration?: number;
```

Comment:
Are we using this prop for anything?

Reply:
Oh, good catch, I did use it in one of my tests, but not anymore. I'll remove it.